tests/unittests/Repro.cpp

```diff
@@ -588,6 +593,16 @@ int run() {
 
   std::list<InferenceResult> results;
 
+  if (glowDumpTrace && glowEnableDeviceTrace) {
+    // Start device traces.
+    hostManager->setTraceContext(
+        glow::make_unique<TraceContext>(TraceLevel::STANDARD));
+    Error err = hostManager->startDeviceTrace();
```

Comment:
This will leave the err unchecked. It needs to be std::moved to one of  ERR_TO_STRING, ERR_TO_BOOL, or ERR_TO_VOID. ERR_TO_STRING and logged below would be useful